apps/src/netsim/NetSimVisualization.js

```diff
@@ -143,10 +143,11 @@ NetSimVisualization.prototype.tick = function (clock) {
 
 /**
  * Render: Let all vizentities "redraw" (or in our case, touch the DOM)
+ * @param {RunLoop.Clock} clock
  */
-NetSimVisualization.prototype.render = function () {
+NetSimVisualization.prototype.render = function (clock) {
```

Comment:
It gets attached to the `RunLoop.render` event [here](https://github.com/code-dot-org/code-dot-org/pull/2998/files#diff-ab3ea858802e4dc9540752530316fb97R119), which [always passes the clock as an argument](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/RunLoop.js#L105) - it just wasn't being used in this method before.